src/main/scala/com/microsoft/hyperspace/Hyperspace.scala

```diff
@@ -36,7 +36,7 @@ class Hyperspace(spark: SparkSession) {
   /**
    * Create index.
    *
-   * @param df the DataFrame object to build index on.
+   * @param df          the DataFrame object to build index on.
```

Comment:
revert?

Reply:
Hm, interesting, this change was introduced by scalafmt, I didn't even touch this. Let me see why it is doing that. Sorry about the formatting trouble.

Reply:
I have manually resolved the formatting, however it seems scalafmt is consistently assigning tab spaces for parameter definitions to align content in a tabular format.

Reply:
At this point I don't know what specific standard we are following or am I using scalafmt in an incorrect manner, any help in this would be appreciated. Thank you.

Reply:
@apoorvedave1 helped me out, I will update the doc later it wasn't straightforward for me. Thanks !